### ISSUE TITLE
OH GOD A BUG

### ISSUE THREAD

**Issue by TRRESTGHGYTCFTGBCXSZD** [sk7725/BetaMindy#31]
**Platform**: Windoes

**Build**: Foo's Client Build 84

**Issue**: Trying To Load Betamindy in Foo's Client Crashes Game

**Steps to reproduce**: Run Betamindy With Foo's Client

**Last log**:
java.lang.NoSuchFieldError: graphics
	at betamindy.content.SettingAdder.addGraphicSetting(SettingAdder.java:8)
	at betamindy.content.SettingAdder.init(SettingAdder.java:15)
	at betamindy.BetaMindy.lambda$new$1(BetaMindy.java:40)
	at arc.Events.lambda$fire$2(Events.java:30)
	at arc.struct.Seq.each(Seq.java:200)
	at arc.Events.fire(Events.java:30)
	at arc.Events.fire(Events.java:25)
	at mindustry.ClientLauncher.update(ClientLauncher.java:150)
	at arc.backend.sdl.SdlApplication.listen(SdlApplication.java:158)
	at arc.backend.sdl.SdlApplication.loop(SdlApplication.java:146)
	at arc.backend.sdl.SdlApplication.<init>(SdlApplication.java:43)
	at mindustry.desktop.DesktopLauncher.main(DesktopLauncher.java:57)
```

**Crash logs**:
java.lang.NoSuchFieldError: graphics
	at betamindy.content.SettingAdder.addGraphicSetting(SettingAdder.java:8)
	at betamindy.content.SettingAdder.init(SettingAdder.java:15)
	at betamindy.BetaMindy.lambda$new$1(BetaMindy.java:40)
	at arc.Events.lambda$fire$2(Events.java:30)
	at arc.struct.Seq.each(Seq.java:200)
	at arc.Events.fire(Events.java:30)
	at arc.Events.fire(Events.java:25)
	at mindustry.ClientLauncher.update(ClientLauncher.java:150)
	at arc.backend.sdl.SdlApplication.listen(SdlApplication.java:158)
	at arc.backend.sdl.SdlApplication.loop(SdlApplication.java:146)
	at arc.backend.sdl.SdlApplication.<init>(SdlApplication.java:43)
	at mindustry.desktop.DesktopLauncher.main(DesktopLauncher.java:57)
```

---

*Place an X (no spaces) between the brackets to confirm that you have read the line below.*  
- [X] **I have searched the closed and open issues to make sure that this problem has not already been reported.**
- [X] **I have updated to the latest release (https://github.com/sk7725/BetaMindy/releases) to make sure my issue has not been fixed.**

---

The original template is by Anuke, modified by sk7725.

**Comment by sk7725** [comment]
Foo's client is seriously outdated, so it will not work.

**Comment by buthed010203** [comment]
> Foo's client is seriously outdated, so it will not work.

@sk7725 Nah, it's up to date with mindustry (124.1) but mods just don't like to function for whatever reason and me and foo don't have the time to find out why lol

**Comment by sk7725** [comment]
> > Foo's client is seriously outdated, so it will not work.
> 
> @sk7725 Nah, it's up to date with mindustry (124.1) but mods just don't like to function for whatever reason and me and foo don't have the time to find out why lol

addGraphicSetting() was added (rather, updated to public) in v122.2. The fact that your client still treats this as private means that your client is outdated, at least before 122.2.
It is DEFINITELY not up to date.

**Comment by sk7725** [comment]
TL;DR: Your client is indeed outdated, go tell your foo guy to update all sources up to date, not just random ones so it "looks like 124" lol
